widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -38,99 +38,6 @@
-     * The remote configurations approach is more versatile and better suited for future development.
-     */
-    @Deprecated
-    public static final String UI_THEME = "ui_theme";
-    /**
-     * Use with {@link android.os.Bundle} to pass in the name of your company as a navigation
-     * argument when navigating to {@link com.glia.widgets.chat.ChatActivity}
-     *
-     * @deprecated Use {@link com.glia.widgets.GliaWidgetsConfig.Builder#companyName}
-     * or customize the strings from GliaHub
-     */
-    @Deprecated
-    public static final String COMPANY_NAME = "company_name";
-
-    /**
-     * Use with {@link android.os.Bundle} to pass in the ID of the queue you wish to enroll in
-     * as a navigation argument when navigating to {@link com.glia.widgets.chat.ChatActivity}
-     * or {@link com.glia.widgets.call.CallActivity}
-     *
-     * @deprecated Use {QUEUE_IDS} instead.
-     */
-    @Deprecated
-    public static final String QUEUE_ID = "queue_id";
-
-    /**
-     * Use with {@link android.os.Bundle} to pass in an {@link java.util.ArrayList} of {@link String} of the queues IDs
-     * you wish to enroll in as a navigation argument when navigating to {@link com.glia.widgets.chat.ChatActivity}
-     * or {@link com.glia.widgets.call.CallActivity}
-     */
-    public static final String QUEUE_IDS = "queue_ids";
-    /**
-     * Use with {@link android.os.Bundle} to pass in a context url as a navigation
-     * argument when navigating to {@link com.glia.widgets.chat.ChatActivity}
-     *
-     * @deprecated Use {@link com.glia.widgets.GliaWidgets#CONTEXT_ASSET_ID}
-     */
-    @Deprecated
-    public static final String CONTEXT_URL = "context_url";
-    /**
-     * Use with {@link android.os.Bundle} to pass in a context asset ID as a navigation
-     * argument when navigating to {@link com.glia.widgets.chat.ChatActivity}
-     */
-    public static final String CONTEXT_ASSET_ID = "context_asset_id";
-    /**
-     * It's recommended to use {@link GliaWidgetsConfig.Builder#setUseOverlay(boolean)} ()} instead of this constant directly.
-     * Use with {@link android.os.Bundle}  to pass in a boolean which represents if you would like to use the chat head bubble
-     * as an overlay outside your application for navigating to {@link com.glia.widgets.chat.ChatActivity}.
-     * If set to true then the SDK will ask for overlay permissions and try to always show the navigation bubble outside
-     * the application. However, it will be shown only if the user has accepted the permissions.
-     * If false, then overlay permission is not requested and the navigation bubble is shown when the application is active.
-     * Default value is true.
-     *
-     * @deprecated Use {@link com.glia.widgets.GliaWidgetsConfig#isEnableBubbleOutsideApp() and
-     * {@link GliaWidgetsConfig#isEnableBubbleInsideApp()}
-     */
-    @Deprecated
-    public static final String USE_OVERLAY = "use_overlay";
```

Comment:
Remove of depricated properties should be a separate commit and even separate PR

Reply:
I removed all the properties that are `useless` after these PR changes because they’re mainly used for intent extras that we closed from integrators, so there aren’t only the deprecated ones.
This is part of the breaking changes we will introduce in version 3.0.0. Do you think we need to somehow separate deprecated ones?